crates/oxc_linter/src/snapshots/no_dynamic_delete.snap

```diff
@@ -0,0 +1,92 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ typescript-eslint(no-dynamic-delete): Do not delete dynamically computed property keys.
+   ╭─[no_dynamic_delete.tsx:3:17]
+ 2 │             const container: { [i: string]: 0 } = {};
+ 3 │             delete container['aa' + 'b'];
```

Comment:
consider covering either the entire statement or just the computed key.

Reply:
fixed it by covering the entire statement